examples/language-modeling/README.md

```diff
@@ -574,6 +574,46 @@ python3 ../gaudi_spawn.py --use_deepspeed  --world_size 8  run_lora_clm.py \
   --flash_attention_causal_mask True
 ```
 
+- Multi-card finetuning of Llama2-70B with DeepSpeed ZeRO-3 optimization, LoRA and FP8 precision:
+
+  > The following command requires Habana DeepSpeed 1.13.0 or later.
+
+```bash
+PT_HPU_MAX_COMPOUND_OP_SIZE=10 \
```

Comment:
why do we still need PT_HPU_MAX_COMPOUND_OP_SIZE=10 ?

Reply:
There is a spike in the loss curve when the flag is removed, which results in a slight drop in overall accuracy for Llama2-70B.

So for now we want to keep this flag, while we investigate and fix this issue.